### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by deepeshaburse**
This PR updates all the links not working and renames "Tutorial" to "tutorial" for uniformity.

### CODE REVIEW

**Review by TerryHowe — DISMISSED** [review]
/lgtm

**Comment by TerryHowe** [comment]
Is there any way to validate links during the build of docusaurus? Hugo does that and you avoid problems like this.

**Comment by deepeshaburse** [comment]
The link for the installation guides will work once my previous PR gets merged. As I have shifted the guide to top level.

File: docs/Tutorials/zot-registry-tutorial.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ OCI registries are like a single location where container images and other artif
 
 We will be using zot registry in this guide. Zot registry is an OCI-native container registry for distributing container images and OCI artifacts.
 
-In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](https://oras.land/docs/CLI/installation) to do so. 
+In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](../installation.mdx) to do so.
```
Comment (asmitbm) [inline docs/Tutorials/zot-registry-tutorial.mdx:14]:
```suggestion
In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](/docs/how_to_guides/installation) to do so. 
```

File: docs/index.mdx [hunk 2]
```diff
@@ -11,7 +11,7 @@ Registries are evolving as generic artifact stores.
 To enable this goal, the ORAS project provides a way to
 push and pull OCI Artifacts to and from OCI Registries.
 
-Users seeking a generic registry client can benefit from the [ORAS CLI](./cli/installation), while
+Users seeking a generic registry client can benefit from the [ORAS CLI](./installation.mdx), while
```
Comment (asmitbm) [inline docs/index.mdx:14]:
```suggestion
Users seeking a generic registry client can benefit from the [ORAS CLI](./how_to_guides/installation), while
```

File: docs/index.mdx [hunk 3]
```diff
@@ -67,7 +67,6 @@ application/vnd.unknown.config.v1+json
 Authors of new OCI Artifacts are thus encouraged to define their own media types specific to
 their artifact, which their custom client(s) know how to operate on.
 
-If you wish to start publishing OCI Artifacts right away, take a look at the [ORAS CLI](./cli/installation).
+If you wish to start publishing OCI Artifacts right away, take a look at the [ORAS CLI](./installation.mdx).
```
Comment (asmitbm) [inline docs/index.mdx:70]:
```suggestion
If you wish to start publishing OCI Artifacts right away, take a look at the [ORAS CLI](./how_to_guides/installation).
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -149,12 +149,16 @@ const config = {
                 to: '/docs/',
               },
               {
-                label: 'CLI',
-                to: '/docs/cli/installation',
+                label: 'Installation',
+                to: '/docs/installation',
```
Comment (asmitbm) [inline docusaurus.config.js:153]:
```suggestion
                to: '/docs/how_to_guides/installation',
```

File: src/components/Header/index.jsx [hunk 5]
```diff
@@ -10,7 +10,7 @@ export default function Header() {
                 <h1>ORAS</h1>
                 <h3>Distribute Artifacts Across OCI Registries With Ease</h3>
                 <div className={styles.header_content_input}>
-                    <Link to="/docs/cli/installation" className='button button--secondary button--lg'>Get Started</Link>
+                    <Link to="/docs/installation" className='button button--secondary button--lg'>Get Started</Link>
```
Comment (asmitbm) [inline src/components/Header/index.jsx:13]:
```suggestion
                    <Link to="/docs/how_to_guides/installation" className='button button--secondary button--lg'>Get Started</Link>

```

File: src/components/ORAS_CLI/index.jsx [hunk 6]
```diff
@@ -28,7 +28,7 @@ export default function ORASCLI() {
         <div className={[styles.orascli, styles.section_padding].join(' ')}>
             <div className={styles.orascli_content}>
                 <h1>Install ORAS CLI in seconds</h1>
-                <p>You can install ORAS CLI on different systems or set up in GitHub Actions in just a few seconds. See more <a href="/docs/cli/installation">installation methods</a>.</p>
+                <p>You can install ORAS CLI on different systems or set up in GitHub Actions in just a few seconds. See more <a href="/docs/installation">installation methods</a>.</p>
```
Comment (asmitbm) [inline src/components/ORAS_CLI/index.jsx:31]:
```suggestion
                <p>You can install ORAS CLI on different systems or set up in GitHub Actions in just a few seconds. See more <a href="/docs/how_to_guides/installation">installation methods</a>.</p>

```

**Review by asmitbm — CHANGES_REQUESTED** [review]
We can avoid using .mdx extension to maintain consistency, it doesn't matter if the file extension is present in the link.

**Comment by asmitbm** [comment]
> The link for the installation guides will work once my previous PR gets merged. As I have shifted the guide to top level.

Oh just noticed this right now

**Comment by asmitbm** [comment]
> Is there any way to validate links during the build of docusaurus? Hugo does that and you avoid problems like this.

There is actually, but it is visible on Netlify's end [Line 91](https://app.netlify.com/sites/oras-project/deploys/64aeb898836fb80008ce2881#L91). I will check if that can be added in this CI.

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/Tutorials/zot-registry-tutorial.mdx [hunk 7]
```diff
@@ -11,7 +11,7 @@ OCI registries are like a single location where container images and other artif
 
 We will be using zot registry in this guide. Zot registry is an OCI-native container registry for distributing container images and OCI artifacts.
 
-In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](https://oras.land/docs/CLI/installation) to do so. 
+In order to follow the steps given, you would be required to install the ORAS CLI. You can follow the [installation guide](/docs/installation) to do so.
```
Comment (sajayantony) [inline docs/Tutorials/zot-registry-tutorial.mdx:14]: Looks like we need to first merge  - https://github.com/oras-project/oras-www/pull/216 as per @TerryHowe

Comment (TerryHowe) [reply docs/Tutorials/zot-registry-tutorial.mdx:14]: merged

Comment (deepeshaburse) [reply docs/Tutorials/zot-registry-tutorial.mdx:14]: @TerryHowe @sajayantony all the links work now. Sorry for the inconvenience!

**Review by sajayantony — APPROVED** [review]

**Pull request merged**